todo_app/templates/index.html

```diff
@@ -3,16 +3,37 @@
 
 {% block content %}
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="display-4">TASKMASTER</h1>
+    <p class="lead">All the information is on the task!</p>
   </div>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
-    </div>
-  </div>
+  <h2>My Tasks</h2>
+    <form method="POST">
+            {% for item in items %}
+              {% if item.status == 'Done' %}
+                <input type="checkbox" id="action" name="action" value={{item.id}} class="form-check-input">
+                <label for="action" style="color:grey" class="form-check-label">{{item.title}}</label> 
+                <span class="badge bg-primary rounded-pill">DONE</span><br>
+              {% else %}
+                <input type="checkbox" id="action" name="action" value={{item.id}} class="form-check-input">
+                <label for="action" style="color:black" class="form-check-label"> {{item.title}}</label><br>
+              {% endif %}
+            {% endfor %}
+            <br>
+      <button name="action" class="btn btn-primary mb-3" formaction="/done" formmethod="POST">Done</button>
+      <button name="action" class="btn btn-primary mb-3" formaction="/delete" formmethod="POST">Delete</button>
```

Comment:
I think these button names are why you're finding you're passing through an extra "action" element - is there any reason not to clear these names or at least rename them "done-button" and "delete-button" or similar so they don't clash with the checkboxes?
E.g. if we do the following, then our loops can just be `for item in done_items:` rather than ... `done_items[0:-1]`
```suggestion
      <button class="btn btn-primary mb-3" formaction="/done" formmethod="POST">Done</button>
      <button class="btn btn-primary mb-3" formaction="/delete" formmethod="POST">Delete</button>
```